guides/common/modules/con_foreman-on-aws-rh-cloud-prerequisites.adoc

```diff
@@ -0,0 +1,11 @@
+= Red Hat Cloud prerequisites
+
+To use this guide, you must complete the following steps:
+
+* Register with Red Hat Cloud Access.
```

Comment:
This might be outside the scope of your PR but I don't think we should refer to these items as prerequisites. I would be inclined to turn this into a note along these lines:

You must use Red Hat Cloud subscriptions and certified images. For more information, see ...

Reply:
I assume that this is Satellite-specific. -> no change for now, but would apply suggestions/patches.